src/components/ImmutableDevices/helpers.js

```diff
@@ -12,11 +12,13 @@ import {
   TimesCircleIcon,
   UnknownIcon,
 } from '@patternfly/react-icons';
-import dangerColor from '@patternfly/react-tokens/dist/esm/global_danger_color_100';
-import warningColor from '@patternfly/react-tokens/dist/esm/global_warning_color_100';
-import successColor from '@patternfly/react-tokens/dist/esm/global_success_color_100';
-import infoColor from '@patternfly/react-tokens/dist/esm/global_info_color_100';
-import activeColor from '@patternfly/react-tokens/dist/esm/global_active_color_100';
+import {
+  dangerColor,
+  successColor,
+  warningColor,
+  infoColor,
+  activeColor,
+} from '@patternfly/react-tokens';
```

Comment:
It seems that webpack now complain about these. I think the imports are named like the filenames before, like `global_success_color_100` and `global_warning_color_100`.

Reply:
If I dont use this current path, the only other path I can find is the old `dist/esm/colorNameHere`

Reply:
no, no. what I mean is that the import names, like `activeColor` or `successColor` have to be something like `global_success_color_100`. the `from '@patternfly/react-tokens'` is correct.

If you look at the Terminal when webpack compiled, you can see a wall of possible exports from '@patternfly/react-tokens'.